Front end build in a CI environment causes build failure

When running a `dotnet build` in a CI environment the pre-build event calls `npm run build` but it doesn't ensure an `npm install` has already been run. This will cause the NPM script to fail and the therefore the `dotnet build` will fail.

Log output:

```> <themename>@0.0.1 build D:\a\1\s\src\Themes\<themename>
  > npm run bundle:prod && npm run build:patterns && npm run copy:patterns
  
  
  > <themename>@0.0.1 bundle:prod D:\a\1\s\src\Themes\<themename>
  > webpack -p
  
  C:\npm\prefix\node_modules\webpack-cli\bin\cli.js:231
  				throw err;
  				^
  
EXEC : error : Cannot find module 'autoprefixer' [D:\a\1\s\src\Themes\<themename>\<themename>.csproj]```

I propose that the `build` npm script also calls `npm install` first.

--- a/content/package.json
+++ b/content/package.json
@@ -5,7 +5,7 @@
     "author": "Moov2 Ltd.",
     "scripts": {
         "copy:patterns": "copyfiles -u 2 \"PatternLibrary/dist/**/*\" \"wwwroot/patterns\"",
-        "build": "npm run bundle:prod && npm run build:patterns && npm run copy:patterns",
+        "build": "npm install && npm run bundle:prod && npm run build:patterns && npm run copy:patterns",
         "build:patterns": "fractal build",
         "bundle:dev": "webpack -d",
         "bundle:prod": "webpack -p",

